bot/cogs/sudo.py

```diff
@@ -45,27 +42,34 @@ def get_uptime(self) -> str:
         minutes, seconds = divmod(rem, 60)
         days, hours = divmod(hours, 24)
         if days:
-            str = f"{days} days, {hours} hr, {minutes} mins, and {seconds} secs"
+            formatted = f"{days} days, {hours} hr, {minutes} mins, and {seconds} secs"
         else:
-            str = f"{hours} hr, {minutes} mins, and {seconds} secs"
+            formatted = f"{hours} hr, {minutes} mins, and {seconds} secs"
+        return formatted
 
-        return str
+    @staticmethod
+    async def cog_check(ctx: Context) -> t.Union[bool, None]:
+        """Only devs can use this."""
+        if ctx.author.id in config.devs:
+            return True
+        embed = Embed(description="This is an owner-only command, you cannot invoke this.", color=Color.red())
+        await ctx.send(embed=embed)
 
     @group(hidden=True)
     async def sudo(self, ctx: Context) -> None:
         """Administrative information."""
-        pass
 
-    @sudo.command()
+    @sudo.command(aliases=["shutdown", "logout"])
```

Comment:
I'm personally not a huge fan of having`"logout"` as an alias for `shutoff`
```suggestion
    @sudo.command(aliases=["shutdown"])
```

Reply:
Okay. Added it because that's the name for my bot